generators/app/prompts.js

```diff
@@ -25,13 +44,41 @@ module.exports = {
       choices: [
         {
           name: 'factory_girl_rails',
-          checked: true,
+          checked: false,
           value: "gem 'factory_girl_rails'"
         }
       ]
     }]).then(function(answers) {
       this.testGems = answers.gems;
       this.log('\n');
     }.bind(this));
+  },
+
+  env: function() {
+    var defaultEnvFile = ejs.render(this.fs.read(this.templatePath('shared/.env')), {
```

Comment:
Well, the reason I made a setup subgenerator is because yeoman is a billion times more flexible and powerful than a simple js and ruby script, at least if you don't intend to reinvent the wheel.

It's also important to remember that the setup script is **not** something run just when you create the application, but by anyone cloning the project and wanting it ready to work.

Currently, the `shared/.env` file is in the `.gitignore`, which means that it is not supposed to be versioned. But then, anyone cloning the project afterwards would have to create a `.env` by hand, without any kind of hint, specially without the `setup` subgenerator.

Also note that you have "$ node bin/setup.js" on the README.md but the file is named only `setup`.